modules/charon-core/src/main/java/org/wso2/charon3/core/objects/plainobjects/ScimAddress.java

```diff
@@ -0,0 +1,237 @@
+package org.wso2.charon3.core.objects.plainobjects;
+
+/**
+ * author Pascal Knueppel <br>
+ * created at: 19.09.2018 - 09:49 <br>
+ * <br>
+ * this class representation can be used to easily add an address to an
+ * {@link org.wso2.charon3.core.objects.AbstractSCIMObject} object
+ */
+public class ScimAddress {
+
+    /**
+     * formatted The full mailing address, formatted for display or use with a mailing label. This attribute
+     * MAY contain newlines.
+     */
+    private String formatted;
+
+    /**
+     * A label indicating the attribute's function, e.g., "work" or "home".
+     */
+    private String type;
+
+    /**
+     * The full street address component, which may include house number, street name, P.O. box, and
+     * multi-line extended street address information. This attribute MAY contain newlines.
+     */
+    private String streetAddress;
+
+    /**
+     * The city or locality component.
+     */
+    private String locality;
+
+    /**
+     * The state or region component.
+     */
+    private String region;
+
+    /**
+     * The zip code or postal code component.
+     */
+    private String postalCode;
+
+    /**
+     * The country name component. When specified, the value MUST be in ISO 3166-1 "alpha-2" code format
+     * [ISO3166]; e.g., the United States and Sweden are "US" and "SE", respectively.
+     */
+    private String country;
+
+    /**
+     * A Boolean value indicating the 'primary' or preferred attribute value for this attribute, e.g., the
+     * preferred mailing address or the primary email address. The primary attribute value "true" MUST appear
+     * no more than once. If not specified, the value of "primary" SHALL be assumed to be "false".
+     */
+    private boolean primary;
+
+    public ScimAddress() {
```

Comment:
We could remove the constructor as it is default.

Reply:
this might become a problem because I defined an empty constructor and a constructor containing all parameters